src/lang/cpp/internal/BindingsResolver.java

```diff
@@ -840,6 +840,16 @@ public ISourceLocation resolveBinding(IASTNameOwner node, ISourceLocation origin
 		return failedBinding("unknown");
 	}
 
+	public ISourceLocation resolveBinding(ICPPBinding binding, ISourceLocation origin) {
```

Comment:
Is there a particular reason that the preexisting interface method did not suffice?

Reply:
Yes, due to multiple inheritance of interfaces, the code for reaching ICPPBinding was not always reachable, in particular when the object did not accidentally also implement IASTNameOwner.